lessons/12-css-pseudo-classes/index.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+    
+        <style>
+        li:hover:nth-child(2) {
+            background: red;
+            color: white;
+        }
+        li:first-of-type {
+            font-size: 32px;
+        }
+        li:nth-of-type(3n) {
+            color: green;
+        }
+        </style>
+    <h1>Funky List Items</h1>
+    </head>
+    <body>
+        <title>Pseudo Classes</title>
+        <div>
+        <ul>
```

Comment:
Watch out, the `<h1>` needs to be inside of the body because we want it to be rendered in the browser